libbpf-tools/profile.bpf.c

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2021 Wenbo Zhang
```

Comment:
Wrong copyright info. Please use C++ style comments.

Reply:
I edited comment style.
For copyright information, I've now followed opensnoop's license-identifier.
I hope to have one chance to fix it after more code review.
Thank you.